modules/aggregation-layers/package.json

```diff
@@ -38,18 +38,16 @@
     "prepublishOnly": "npm run build-bundle && npm run build-bundle -- --env=dev"
   },
   "dependencies": {
-    "@babel/runtime": "^7.0.0",
     "@luma.gl/constants": "9.0.0-beta.4",
-    "@luma.gl/core": "9.0.0-beta.4",
     "@luma.gl/shadertools": "9.0.0-beta.4",
     "@math.gl/web-mercator": "^4.0.0",
     "d3-hexbin": "^0.2.1"
   },
   "peerDependencies": {
     "@deck.gl/core": "^9.0.0-alpha",
     "@deck.gl/layers": "^9.0.0-alpha",
-    "@luma.gl/core": "9.0.0-beta.4",
-    "@luma.gl/engine": "9.0.0-beta.4"
+    "@luma.gl/core": "^9.0.0-beta",
```

Comment:
Users don't have to explicitly specify luma deps right? they are covered by deck.gl/core or something? Why is shadertools handled differently?

Reply:
Technically the only package that has to go into peer dep is core, which does not allow multiple copies.